### PR TITLE
Pmidc ginni finance pi 9074

### PR DESCRIPTION
**Pull request by egovmis1**

### CODE REVIEW

File: egov/egov-egfweb/src/main/java/org/egov/egf/web/actions/voucher/PreApprovedVoucherAction.java [hunk 1]
```diff
@@ -160,6 +160,7 @@
     private PersistenceService persistenceService;
     @Autowired
     private EgovCommon egovCommon;
+    private CityService cityService;
```
Comment (subhash-eGov) [inline egov/egov-egfweb/src/main/java/org/egov/egf/web/actions/voucher/PreApprovedVoucherAction.java:163]: Remove this unused code

File: egov/egov-egfweb/src/main/java/org/egov/egf/web/actions/voucher/PreApprovedVoucherAction.java [hunk 2]
```diff
@@ -521,9 +522,17 @@ public String loadvoucherview() throws ApplicationException {
                     Long.valueOf(parameters.get(VHID)[0]));
             from = FinancialConstants.STANDARD_VOUCHER_TYPE_JOURNAL;
         }
-        heading = ReportUtil.getCityName();
+       //heading = ReportUtil.getCityName();
+       heading = microserviceUtils.getHeaderNameForTenant().toUpperCase();
         getMasterDataForBillVoucher();
         getHeaderMandateFields();
+
```
Comment (subhash-eGov) [inline egov/egov-egfweb/src/main/java/org/egov/egf/web/actions/voucher/PreApprovedVoucherAction.java:529]: Remove this as well

**Review by subhash-eGov — CHANGES_REQUESTED** [review]

File: egov/egov-egfweb/src/main/java/org/egov/egf/web/actions/voucher/PreApprovedVoucherAction.java (hunk 1 above)
Comment (egovmis1) [reply egov/egov-egfweb/src/main/java/org/egov/egf/web/actions/voucher/PreApprovedVoucherAction.java]: removed

File: egov/egov-egfweb/src/main/java/org/egov/egf/web/actions/voucher/PreApprovedVoucherAction.java (hunk 2 above)
Comment (egovmis1) [reply egov/egov-egfweb/src/main/java/org/egov/egf/web/actions/voucher/PreApprovedVoucherAction.java:527]: removed

**Review by subhash-eGov — APPROVED** [review]

**Pull request merged**